frontend/plan/pages/index.tsx

```diff
@@ -305,6 +332,9 @@ function Index() {
                         <div style={{ padding: "10px" }}>
                             <Cart setTab={setTab} mobileView={true} />
```

Comment:
Bad prop name; use onChange, don't hardcode `mobileView`, could be a context or a hook to decide if user is using mobile view